draft-scenarios/clash/gold_rush.tex

```diff
@@ -17,123 +17,112 @@ \subsection*{\MakeUppercase{Player Setup}}
 
 \textbf{Player Count:} 1, 2 or 4 (Alliance variant)
 
-\textbf{Starting Resources:} 15 \svg{gold}, 6 \svg{building_materials}, 1 \svg{valuables}
+\textbf{Starting Resources:} 15 \svg{gold}, 3 \svg{building_materials}, 1 \svg{valuables}
 
 \textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}
 
 \textbf{Starting Units:}
 \begin{itemize}
-  \item A Pack of \bronze\ Units with the hightst Recruitment cost
+  \item A Pack of \bronze\ Units with the highest Recruitment cost
   \item A Few \silver\ Units with the lowest Recruitment cost
```

Comment:
```suggestion
  \item A Pack of chosen \bronze\ Units
  \item A Few cheapest \silver\ Units
```
If you feel like it, I'm a fan of @xNeuroNx's convention. But the way it's written right now is fine.